src/SFA.DAS.ApplyService.Application/Apply/IApplyRepository.cs

```diff
@@ -23,7 +24,10 @@ public interface IApplyRepository
         Task<string> GetGatewayPageStatus(Guid applicationId, string pageId);
         Task<string> GetGatewayPageComments(Guid applicationId, string pageId);
 
-        Task SubmitGatewayPageAnswer(Guid applicationId, string pageId, string userId, string userName, string status, string comments);
+        Task InsertGatewayPageAnswer(GatewayPageAnswer pageAnswer, string userId, string userName);
+        Task UpdateGatewayPageAnswer(GatewayPageAnswer pageAnswer, string userId, string userName);
+        Task UpdateApplication(Domain.Entities.Apply application);
```

Comment:
Would it be useful if these return a bool so the caller knows if the insert/update was successful